frontend/src/metabase/search/components/SearchSidebar/DropdownSidebarFilter/DropdownSidebarFilter.tsx

```diff
@@ -2,8 +2,10 @@ import { isEmpty } from "underscore";
 import type { MouseEvent } from "react";
```

Comment:
I rename the `SidebarFilter` type to `DropdownSidebarFilter` to reflect the addition of a `ToggleSidebarFilter` type. Most of these changes are just renaming